src/wp-admin/includes/dashboard.php

```diff
@@ -1561,61 +1561,36 @@ function wp_dashboard_empty() {}
-		<?php else : ?>
-			<li><?php printf( '<a href="%s" class="welcome-icon welcome-write-blog">' . __( 'Write your first blog post' ) . '</a>', admin_url( 'post-new.php' ) ); ?></li>
-			<li><?php printf( '<a href="%s" class="welcome-icon welcome-add-page">' . __( 'Add an About page' ) . '</a>', admin_url( 'post-new.php?post_type=page' ) ); ?></li>
-		<?php endif; ?>
-			<li><?php printf( '<a href="%s" class="welcome-icon welcome-view-site">' . __( 'View your site' ) . '</a>', home_url( '/' ) ); ?></li>
-		</ul>
-	</div>
-	<div class="welcome-panel-column welcome-panel-last">
-		<h3><?php _e( 'More Actions' ); ?></h3>
-		<ul>
-		<?php if ( current_theme_supports( 'widgets' ) || current_theme_supports( 'menus' ) ) : ?>
-			<li><div class="welcome-icon welcome-widgets-menus"><?php
-				if ( current_theme_supports( 'widgets' ) && current_theme_supports( 'menus' ) ) {
-					printf( __( 'Manage <a href="%1$s">widgets</a> or <a href="%2$s">menus</a>' ),
-						admin_url( 'widgets.php' ), admin_url( 'nav-menus.php' ) );
-				} elseif ( current_theme_supports( 'widgets' ) ) {
-					echo '<a href="' . admin_url( 'widgets.php' ) . '">' . __( 'Manage widgets' ) . '</a>';
-				} else {
-					echo '<a href="' . admin_url( 'nav-menus.php' ) . '">' . __( 'Manage menus' ) . '</a>';
-				}
-			?></div></li>
-		<?php endif; ?>
-		<?php if ( current_user_can( 'manage_options' ) ) : ?>
-			<li><?php printf( '<a href="%s" class="welcome-icon welcome-comments">' . __( 'Turn comments on or off' ) . '</a>', admin_url( 'options-discussion.php' ) ); ?></li>
-		<?php endif; ?>
-			<li><?php printf( '<a href="%s" class="welcome-icon welcome-learn-more">' . __( 'Learn more about getting started' ) . '</a>', __( 'https://codex.wordpress.org/First_Steps_With_WordPress' ) ); ?></li>
-		</ul>
-	</div>
-	</div>
+	<p class="about-description"><?php printf( __( 'Thank you for trying ClassicPress&nbsp;%s' ), $display_version ); ?></p>
+			<h3><?php _e( 'Feedback and Support' ); ?></h3>
+			<p>
+				<?php _e(
+					"Do you have any feedback about this version of ClassicPress?"); ?>
+			</p>
+			<p>
+				<?php printf(
+						__(
+							/* translators: link with instructions to join ClassicPress Slack */
+							'For support, suggestions for improvement, or general discussion about ClassicPress, <a href="%s" target="_blank">join our Slack group</a> and send us a message in the <a href="%s" target="_blank"><strong>#support</strong></a>, or <a href="%s" target="_blank"><strong>#testing</strong></a> channels.'
+						),
+						'https://join.slack.com/t/classicpress/shared_invite/enQtNDIwNjY2OTg1MjAxLWJiM2U2NmY3ZjFlZjQ4Zjk2OGI4ZTg3NzY1ZTU3NzI3OTRjMTU0YzAzOWUyZmZlODgyOWE1YTViYjcwY2Y5YzI', 'https://classicpress.slack.com/messages/support/', 'https://classicpress.slack.com/messages/testing/'
+					); ?>
+			</p>
+			<p>
+				<?php printf(
+			__(
```

Comment:
Nitpick: indentation looks off in this block.

Reply:
Adjusted the indent.